frontend/src/views/consumer/paymentstatus-modal/index.js

```diff
@@ -91,23 +91,26 @@ export default withTheme(
                       {longText}
                     </Typography>
                     <br />
-                    {query.status == 4 &&
-                      event && (
-                        <Typography
-                          style={{ whiteSpace: 'pre-line' }}
-                          type="largebody"
-                          color={this.props.theme.palette.primaryDeep}
-                        >
-                          {event.contactInformation}
-                        </Typography>
-                      )}
-                  </Content>
-                  {event &&
-                    query.status == 0 && (
-                      <EventCardWrapper bgColor={event.themeColor}>
-                        <EventCard mini event={event} />
-                      </EventCardWrapper>
+                    {query.status == 0 && event && (
```

Comment:
I would suggest you do it like this on the same line 99 by reusing the code.

```
                  {(query.status == 4 || query.status == 0) && event && (

                      <Typography
                        style={{ whiteSpace: 'pre-line' }}
                        type="largebody"
                        color={this.props.theme.palette.primaryDeep}
                      >
                        {event.contactInformation}
                      </Typography>
                 )}
```

Reply:
👍

Reply:
The color code scheme is already there for status == 4, so it would be better if we use same color coding everywhere for consistency. 

Yes you could use the same typography in reservation-status-modal also
```
<Typography
	                        style={{ whiteSpace: 'pre-line' }}
	                        type="largebody"
	                        color={this.props.theme.palette.primaryDeep}
	                      >
```